Example/SSDynamicTextTests/SSDynamicViewsReleaseTests.m

```diff
@@ -7,70 +7,69 @@
 //
 
 #import <XCTest/XCTest.h>
-#import "SSDynamicText.h"
 #import "SSTestsHelper.h"
-
-/**
- * Class provided to test unregistering of `UIContentSizeCategoryDidChangeNotification` notification.
- * File has disabled ARC via `-fno-objc-arc` compiler flag in compile sources.
- * Please DO NOT add more tests in this file.
- */
+#import "SSDynamicText.h"
 
 @interface SSDynamicViewsReleaseTests : XCTestCase
-
 @end
 
 @implementation SSDynamicViewsReleaseTests
 
-- (void)testDynamicLabelShouldNotObserveTextSizeChangesAfterRelease {
```

Comment:
There's no point o keeping those code any more. According to Apple docs:

> If your app targets iOS 9.0 and later or macOS 10.11 and later, you don't need to unregister an observer in its deallocation method. If your app targets earlier releases, be sure to invoke removeObserver:name:object: before observer or any object specified in addObserver:selector:name:object: is deallocated.

So since iOS 9 we are not able to cause crash by posting notification to released and still observing object, because unregistering comes out of box.
We can't also, create view mock, release them and expect some method, because this will cause false-positive test result.
Any idea?

More: [addobserver](https://developer.apple.com/reference/foundation/nsnotificationcenter/1415360-addobserver?language=objc)